### PR TITLE
Creamy tomato soup, soup-free chili.

### LINKED ISSUE #75886: Chili and Tomato Soup reaction conflict
Chili and Tomato Soup have a conflicting recipe. Attempting to cook Chili in a soup pot will result in half of the reaction being tomato soup if you are making chili with a portion size larger than 1. A similar problem would probbably occur in Chili sin Carne, as it also has a recipie which includes all the ingredients for tomato soup in a large batch.

## Reproduction:

Prepare a pot of chili, using twice the minimum ingredients. 

Observe that your chili is contaminated by tomato soup.

### PR DESCRIPTION
**Pull request by Dorsisdwarf**
i tried to make chili and it made 50% tomato soup immersion RUINED
## About The Pull Request
Adds 5u of cream to the recipe for tomato soup. This means that tomatosoup is no longer produced as an unavoidable byproduct of all soup recipes that include a tomato when making more than one serving.
## Why It's Good For The Game
Fixes #75886 , makes soup tastier
## Changelog
:cl:
fix: Nanotrasen Cookery Command has ended the plague of staff complaining that their tomato dishes are just complicated forms of soup by redefining "tomato soup" to include a dash of cream.
/:cl:

### CODE REVIEW

**Comment by DaydreamIQ** [comment]
Glad im not the only one who had this exact issue. Mah fuckin chilli was RUINED

**Comment by CRITAWAKETS** [comment]
Not every tomato soup uses cream (please help my dad sucks at making tomato soup), since there isn't any basil in the game, I think maybe black pepper could be an interesting ingredient instead?
plus, cream in tomato soup makes it non-vegan, which is kinda a weird property for tomato soup to have

File: code/modules/food_and_drinks/recipes/soup_mixtures.dm [hunk 1]
```diff
@@ -551,7 +551,10 @@
 	drink_type = VEGETABLES | FRUIT // ??
 
 /datum/chemical_reaction/food/soup/tomatosoup
-	required_reagents = list(/datum/reagent/water = 50)
+	required_reagents = list(
+		/datum/reagent/water = 50,
```
Comment (Fikou) [inline code/modules/food_and_drinks/recipes/soup_mixtures.dm:555]:
```suggestion
		/datum/reagent/water = 45,
```

Comment (Dorsisdwarf) [reply code/modules/food_and_drinks/recipes/soup_mixtures.dm:555]: Given that you always put 50u of water in a cookpot when you fill it this is just going to end up with leftover water in the soup mix, but I'll do it.

**Comment by Sealed101** [comment]
don't tell goof

**Pull request merged**